recipes/cerberecontrib_ghrsst/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set python_min = "3.10" %}
+{% set name = "cerberecontrib-ghrsst" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.ifremer.fr/cerbere/cerberecontrib-ghrsst/-/archive/{{ version }}/cerberecontrib-ghrsst-{{ version }}.tar.gz
+  sha256: 7f84aeb91bb181c20b2ebe9f9405407d0f4717a187e66f4366eb4ad49cd970c3
+
+build:
+  noarch: python
+  script_env:
+    - POETRY_DYNAMIC_VERSIONING_BYPASS={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core >=1.0.0
+    - poetry-dynamic-versioning >=1.0.0,<2.0.0
+    - pip
+  run:
+    - python {{ python_min }}
+    - cerbere >=3.1.2,<4.0.0
```

Comment:
The metadata in this package should be fixed and pip-check added there as well to avoid regressions.

```
+ pip check
cerberecontrib-ghrsst 3.1.2 has requirement cerbere<4.0.0,>=3.1.2, but you have cerbere 0.0.0.post21+c94c7d1.
```

Reply:
@ocefpaf Is this an issue coming from the metadata within the pyproject.toml ? or somewhere else ?

Thanks a lot for the help end review !!

Reply:
It depends on how the package creates the version info. Sadly the Python packaging ecosystem has way too many ways to do this. Depending on what you use we can "fake" a version and fix it later upstream, I'll check `cerbere` in a moment.